### ISSUE TITLE
[Bug Report] WARN handlePkt: qName cannot be empty

### ISSUE THREAD

**Issue by jojo147** [daeuniverse/dae#445]
### Checks

- [X] I have searched the existing issues
- [X] I have read the documentation
- [X] Is it your first time sumbitting an issue

### Current Behavior

[Jan 26 16:01:33]  WARN handlePkt: qName cannot be empty
[Jan 26 16:01:33]  WARN handlePkt: qName cannot be empty
[Jan 26 16:01:33]  WARN handlePkt: qName cannot be empty
[Jan 26 16:01:33]  WARN handlePkt: qName cannot be empty
[Jan 26 16:01:33]  WARN handlePkt: qName cannot be empty
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x473c80]

goroutine 2367441 [running]:
errors.(*errorString).Error(0xe03900?)
        errors/errors.go:71
github.com/daeuniverse/dae/control.(*DnsController).dialSend.func2()
        github.com/daeuniverse/dae/control/dns_control.go:597 +0x5a2
created by github.com/daeuniverse/dae/control.(*DnsController).dialSend in goroutine 2366327
        github.com/daeuniverse/dae/control/dns_control.go:584 +0x129f

跑一阵子后就会这样子结束程序

### Environment

- **Dae version (use `dae --version`)**: dae version v0.5.1

- **OS (e.g `cat /etc/os-release`)**: Debian GNU/Linux 10 (buster)
- **Kernel (e.g. `uname -a`)**: Linux debian10A 5.10.0-0.deb10.16-amd64 #1 SMP Debian 5.10.127-2~bpo10+1 (2022-07-28) x86_64 GNU/Linux
- **Others**:

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.
